src/App.js

```diff
@@ -0,0 +1,190 @@
+import React, { useEffect, useState } from 'react';
+import List from './List';
+import Tabs from './components/Tabs';
+import firebase from "firebase/compat";
+import { useCollection } from "react-firebase-hooks/firestore";
+import {generateUniqueID} from "web-vitals/dist/modules/lib/generateUniqueID";
+
+import './styles.css';
+import {filterType, modeType} from './Constants';
+
+const firebaseConfig = {
+  apiKey: "AIzaSyATi5uqr6dkf4iTvqfPHndGwtHXz0O6O-s",
+  authDomain: "lab-90286.firebaseapp.com",
+  databaseURL: "https://lab-90286-default-rtdb.firebaseio.com",
+  projectId: "lab-90286",
+  storageBucket: "lab-90286.appspot.com",
+  messagingSenderId: "782831139249",
+  appId: "1:782831139249:web:2d846e81abb155eeece590",
+  measurementId: "G-YTZ7W2BRD9"
+};
+
+firebase.initializeApp(firebaseConfig);
+const db = firebase.firestore();
+
+function App() {
+  const collection = db.collection('todo_list');
+  const [query, setQuery] = useState(collection);
+
+  const [todoListData, setTodoListData] = useState([]);
```

Comment:
Shouldn't be state information since it's calculated from results of useCollection.